.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Build
+
+on:
+  pull_request:
+  push:
+    branches: ['**']
+    tags: [v*]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and Push Docker Image
+        uses: Dwolla/jenkins-agents-workflow/.github/actions/build@main
+        with:
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+          BASE_TAG: <my-specified-base-image-tag>
+          TAG_NAME: <my-unsupported-tag-name>
+          IMAGE_NAME: dwolla/otel-collector
+    uses: Dwolla/jenkins-agents-workflow/.github/workflows/build-docker-image.yml@main
```

Comment:
Error in Actions:
```
Cannot define both `uses` and `steps` at the same time for the following jobs: build
```
🤔

Reply:
I don't think we need to call specify uses here in order to call the action! If it's helpful, we're using just the `action` (without the workflow) is the `Dwolla/java` image [here](https://github.com/Dwolla/docker-java/blob/main/.github/workflows/ci.yml#L23)!

Reply:
Thanks, I updated using that as an example and it seems to be working